### PR TITLE
Added version tag for docker build and push

### PR DESCRIPTION
**Pull request by SchSeba**
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/ovs-cni/27)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 REGISTRY ?= quay.io/kubevirt
+OVS_CNI_GIT_VERSION ?= latest
```
Comment (phoracek) [inline Makefile]: It is not connected to Git or version, in Docker it's called just "tag" 

IMAGE_TAG

**Comment by phoracek** [comment]
@SchSeba please also add this new variable to devel-guide.md

**Comment by SchSeba** [comment]
@phoracek Done

**Comment by phoracek** [comment]
LGTM, let's try it!

**Comment by phoracek** [comment]
https://jenkins.ovirt.org/job/kubevirt_ovs-cni_standard-on-ghpush/17/

**Comment by phoracek** [comment]
@SchSeba it failed with:

```
14:00:54 [check-merged.el7.x86_64] ./automation/check-merged.sh: line 16: ./hack/get_tag.sh: No such file or directory
14:00:54 [check-merged.el7.x86_64] + export IMAGE_TAG=
14:00:54 [check-merged.el7.x86_64] + IMAGE_TAG=
14:00:54 [check-merged.el7.x86_64] + make docker-build
14:00:54 [check-merged.el7.x86_64] make: *** No rule to make target `docker-build'.  Stop.
```

**Comment by SchSeba** [comment]
Yes can you revert I will update the PR I know the problem

**Comment by phoracek** [comment]
@SchSeba could you revert it or just fix it in the fix PR?

**Pull request merged**